### DISCUSSION TITLE
HousingDB Post-2010 | Q4 2023 Update

### DISCUSSION THREAD

**Discussion by mvsCB** [NYCPlanning/data-engineering#731] [Q&A] [answered]
Hi,

Just wondering when the Q4 2023 update will be released for the HousingDB Post-2010 dataset.

![image](https://github.com/NYCPlanning/data-engineering/assets/150717746/27a07655-9dd2-41f0-a6e7-f076748b861e)

Looking for the support behind "27,972 homes were completed in new buildings in New York City in 2023, including both market-rate and affordable units. " - quoted here: https://storymaps.arcgis.com/stories/1c9138dc24064b2e8142ff156345a719.

**Comment by fvankrieken** [comment]
Hi - Housing Database 23Q4 will be published in the next week, I'll update when that happens or if that timeline changes.

Finn

**Comment by mvsCB** [comment]
Thank you!

Michael von Schledorn
CBRE Investment Management
200 Park Ave | New York, NY 10166
***@***.******@***.***> | C +1 920.583.6608

**Comment by fvankrieken** [comment, marked as answer]
Hi Michael - HousingDB 23Q4 can now be found on Bytes of the Big Apple

https://www.nyc.gov/site/planning/data-maps/open-data/dwn-housing-database.page#housingdevelopmentproject
